app/templates/sites/index.hbs

```diff
@@ -1,19 +1,23 @@
 {{page-title "Vestigingen"}}
 <PageHeader class="au-o-box">
+
   <:title>Vestigingen</:title>
   <:subtitle>{{this.currentSession.group.name}}
     ({{this.currentSession.groupClassificationLabel}})</:subtitle>
   <:action>
-    <AuLink
-      @route="sites.new"
-      @skin="button"
-      @icon="plus"
-      @iconAlignment="left"
-    >
-      Nieuwe vestiging toevoegen
-    </AuLink>
-    {{! <ReportWrongData /> }}
+    {{#if (is-feature-enabled "edit-feature")}}
+      <AuLink
+        @route="sites.new"
+        @skin="button"
+        @icon="plus"
+        @iconAlignment="left"
+      >
+        Nieuwe vestiging toevoegen
+      </AuLink>
+      {{! <ReportWrongData /> }}
```

Comment:
If the feature is not enabled the user should still be able to report the wrong data